tests/lcs/core/math/math.livecodescript

```diff
@@ -918,3 +918,45 @@ repeat with i = 1 to 10
 end repeat
 
 end TestMathArrayWrapArray
+
+on _RandomMaxLimitTestThrows
+
+	local tX, tN2
+
+	put 2^53 into tX
+	
+	repeat with i = 1 to 128
```

Comment:
I suggest putting a comment here - 'This loop computes the next representable double after tX'